### DISCUSSION TITLE
11.0.6.Final tag missing

### DISCUSSION THREAD

**Discussion by CChengjie** [infinispan/infinispan#10278] [Q&A] [answered]
We found that version **11.0.6.Final** was released in Maven, but it was no corresponding tag in github.
**What is the reason for tag's absence?**
is this release between 11.0.5 and 11.0.7?

**Comment by tristantarrant** [comment, marked as answer]
`11.0.6.Final` was wrongly published to Maven. `11.0.7.Final` is the proper release
